packages/stream-metadata/src/opensea.ts

```diff
@@ -45,25 +151,13 @@ export const refreshOpenSea = async ({
 		throw new Error('Space not found')
 	}
 
-	const url = getOpenSeaAPIUrl(logger, space)
-	logger.info({ url, spaceAddress }, 'refreshing openSea')
+	const allMemberTokenIds = await getAllMemberTokenIds(logger, space)
+	const promises = [
+		refreshSpaceOwnerNft(logger, space),
+		...allMemberTokenIds.map((tokenId) => refreshMemberNft(logger, space, tokenId)),
+	]
```

Comment:
if a town has 1000 members, are we sending 1000 invalidations? i'm curious to see if opensea offers an alternative. could we maybe refresh the entire contract with one request?

Reply:
i'm worried that 1) this could time out and 2) we get rate limited.
open sea provides an on chain even to refresh a collection. seems like they should allow us to do it at the api level too